### PR TITLE
Raw rq

### PR DESCRIPTION
**Pull request by ahmad-musleh**

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -7,6 +8,7 @@ deps=
     pytest
     pytest-cov
     mock
+    redis==2.10.6
```
Comment (hammady) [inline tox.ini:10]: Do we need this now after enabling `requirements.txt`?

File: tox.ini [hunk 2]
```diff
@@ -1,4 +1,5 @@
 [tox]
+; envlist=pypy
```
Comment (hammady) [inline tox.ini]: Is this a comment?

Comment (ahmad-musleh) [reply tox.ini]: This comment if for when we want to test with only the pypy service.

File: tox.ini (hunk 1 above)
Comment (ahmad-musleh) [reply tox.ini:10]: This is for local tox testing. Yes.

**Review by hammady — APPROVED** [review]

**Pull request merged**